backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/readers/CsvTableReader.java

```diff
@@ -77,7 +75,15 @@ public boolean hasNext() {
             }
 
             public Row next() {
-              return new Row(it.next());
+              HashMap<String, String> next = (HashMap<String, String>) it.next();
+              boolean isEmpty = next.values().stream().allMatch(Objects::isNull);
+
+              if (isEmpty) {
+                // skip empty lines
+                return new Row(it.next());
```

Comment:
Shouldn't you recurse instead of calling it.next() otherwise you don't handle two consecutive empty lines.

Reply:
This is indeed pragmatic. However, I think you should consider multiple empty lines, so it should be a 'while'?